test/logged_smtp_delivery_test.rb

```diff
@@ -57,7 +57,13 @@ def without_file_logger
 
     it 'logs the mail to a file when the mail file logger is available' do
       TestMailer.welcome.deliver_now
-      assert_equal ["Date: Sat, 01 Jan 2000 00:00:00 +0000\r\nFrom: me@example.com\r\nTo: to@example.com\r\nMessage-ID: 12345@example.com\r\nSubject: Welcome\r\nMime-Version: 1.0\r\nContent-Type: text/plain;\r\n charset=UTF-8\r\nContent-Transfer-Encoding: 7bit\r\n\r\nhello"], mail_file_logger.messages
+
+      message_id = "12345@example.com"
+      if Gem::Version.new(Mail::VERSION::STRING) >= Gem::Version.new("2.8.0")
+        message_id = "<#{message_id}>"
+      end
+
+      assert_equal ["Date: Sat, 01 Jan 2000 00:00:00 +0000\r\nFrom: me@example.com\r\nTo: to@example.com\r\nMessage-ID: #{message_id}\r\nSubject: Welcome\r\nMime-Version: 1.0\r\nContent-Type: text/plain;\r\n charset=UTF-8\r\nContent-Transfer-Encoding: 7bit\r\n\r\nhello".encode('ASCII-8BIT')], mail_file_logger.messages
```

Comment:
@zdAlexM @alduethadyn 👀

Reply:
those both look legit to me. `ASCII-8BIT` really should have been called `BINARY` but that's only its alias. :(

Reply:
Ah, never mind the encoding. The encoding specific message only shows up in case of spec failure. Since the specs pass (with the `<>` wrapper) I don’t think it’s necessary to add the `#encode` call.